src/Transition/InputProcessor.f90

```diff
@@ -4814,6 +4814,28 @@ FUNCTION IPTrimSigDigits(IntegerValue) RESULT(OutputString)
 
 END FUNCTION IPTrimSigDigits
 
+FUNCTION GetFieldOrIDDDefault(InArgString, FldDefaultString)  RESULT (ResultString)
+
+          ! PURPOSE OF THIS SUBROUTINE:
+          ! If InArgString is Blank, replace with FldDefaultString
+
+  IMPLICIT NONE    ! Enforce explicit typing of all variables in this routine
```

Comment:
Implicit None is already declared at the top of the module, so probably not necessary, but totally fine.